corehq/motech/repeaters/models.py

```diff
@@ -1250,6 +1250,7 @@ class SQLRepeatRecord(models.Model):
     state = models.PositiveSmallIntegerField(choices=State.choices, default=State.Pending)
     registered_at = models.DateTimeField()
     next_check = models.DateTimeField(null=True, default=None)
+    max_possible_tries = models.IntegerField(default=MAX_BACKOFF_ATTEMPTS)
```

Comment:
I understand that this property is added as it was there before in RepeatRecord class. 
Can you explain how do you see this value being used with RepeatRecords and how it will be configured in each repeat record?

Reply:
Normally the max number of tries is a constant, but it will be increased by `MAX_BACKOFF_ATTEMPTS` each time a record is "requeued". That logic is coming in a follow-up PR. It resolves [SC-3158](https://dimagi-dev.atlassian.net/browse/SC-3158).